### ISSUE TITLE
E2E Encrypted VoIP / Video

### ISSUE THREAD

**Issue by jakewb-b** [vector-im/roadmap#28]
Summary
Deliver E2E VoIP voice/video calling over Matrix - delivered initially as an SPA, with potential for future integration into Element clients.

Intended Outcome
E2E encrypted voice/video calls that meet the requirements of a cross-section of different users, including suitability for different types and sizes of meetings.

**Comment by PxBCPG** [comment]
Aren't p2p calls already encrypted ?

**Comment by SimonBrandner** [comment]
> Aren't p2p calls already encrypted ?

Yes, though this issue is specifically about group calls which are done through Jitsi atm

**Comment by weddige** [comment]
Is there a roadmap for the new group call experience? I would love to see it in production.
